xml_inscriptions/transcribed/IL.Chi.AIC.L.#2024.07.01-mdn.xml

```diff
@@ -0,0 +1,233 @@
+        <!--  *************************************  <encodingDesc>  ********************************* -->
+
+        <encodingDesc>
+
+                <xi:include
+                    href="http://library.brown.edu/usep_data/resources/include_taxonomies.xml">
+                    <xi:fallback>
+                        <p>Taxonomies for US Epigraphy controlled values</p>
+                    </xi:fallback>
+                </xi:include>
+
+        </encodingDesc>
+
+        <profileDesc> <!-- not used --> </profileDesc>
+
+
+        <!--  *************************************  <revisionDesc>  ********************************* -->
+    	<!-- Record all changes here -->
+
+        <revisionDesc>
+            <change when="2024-09-22" who="Advaith Rege">Initial Encoding</change>
+        </revisionDesc>
+
+    </teiHeader>
+
+    <!--  *************************************  <facsimile>  *********************************  -->
+        <!-- Used to record images of an inscription.   -->
+
+    <facsimile>
+        <surface>
+            <desc/>
+            <graphic url="IL.Chi.AIC.L.#2024.07.01-mdn-entire.jpg"/>
+        </surface>
+        <surface>
+            <desc>Detail of inscription</desc>
+            <graphic url="IL.Chi.AIC.L.#2024.07.01-mdn-det.jpg"/>
+        </surface>
+    </facsimile>
+
+    <text>
+        <body>
+            <div type="edition">
+            <ab>
+                <lb n="1"/>
+                <expan><abbr>D</abbr><ex>is</ex></expan>
+                <expan><abbr>M</abbr><ex>anibus</ex></expan>.
+                <lb n="2"/>
+                <persName>
+                    <name type="praenomen" key="Marcus">
+                        <expan><abbr>M</abbr><ex>arcus</ex></expan>
+                    </name>
+                    <name type="gentilicium" key="Ulpius">Ulpius</name>
+                    <name type="cognomen" key="Cerdo">Cerdo</name>
+                </persName>
+                <lb n="3"/>titulum posuit
+                <persName>
+                    <name type="gentilicium" key="Claudia">Claudiae</name>
+                    <name type="cognomen" key="Tyche">Tycheni</name>
+                </persName>,
+                <lb n="4"/>coniugi
+                <choice><corr>c</corr><sic>k</sic></choice>arissimae,
+                <lb n="5"/>cum qua
+                <expan><abbr>vix</abbr><ex>it</ex></expan> annos
+                <num value="4">IV</num>,
+                <expan><abbr>mens</abbr><ex>es</ex></expan>
+                <num value="6">VI</num>,
+                die<choice><corr>s</corr><sic><expan><abbr>b</abbr><ex>us</ex></expan></sic></choice>
+                <num value="3">III</num>,
+                <expan><abbr>hor</abbr><ex>as</ex></expan>
+                <num value="10">X</num>.
+                <lb n="6"/>in
+                <expan><abbr>die</abbr><ex>m</ex></expan> mortis gratias maximas egi
+                <lb n="7"/>
+                apu<choice><corr>d</corr><sic>t</sic></choice> deos et
```

Comment:
Again, this is a spelling that is not uncommon in the imperial period, so I'm not sure that we'd need to correct it to the more standard orthography (unless it doesn't align with what is on the original, in which case the correction makes good sense; this applies to my other comments above too).